src/depackers/readhuff.c

```diff
@@ -38,7 +38,7 @@ static int rawinput(struct data_in_out *io)
 	return -1;
 }
 
-static void rawoutput(int byte, struct data_in_out *io)
+static int rawoutput(int byte, struct data_in_out *io)
```

Comment:
This doesn't return anything

Reply:
Thanks, fixed. I missed it because libxmp doesn't actually use this file...